bin/deploy

```diff
@@ -7,16 +7,16 @@ SCRIPT_DIR=$(cd "$(dirname "$0")"; pwd)
 # shellcheck disable=SC1090
 source "${SCRIPT_DIR}/../etc/settings.cfg.sh"
 
-EUPS_VERSION=${EUPS_VERSION:-2.1.4}         # Version of EUPS to install
-EUPS_GITREV=${EUPS_GITREV:-""}
-EUPS_GITREPO=${EUPS_GITREPO:-https://github.com/RobertLuptonTheGood/eups.git}
+LSST_EUPS_VERSION=${LSST_EUPS_VERSION:-2.1.4}
+LSST_EUPS_GITREV=${LSST_EUPS_GITREV:-""}
+LSST_EUPS_GITREPO=${LSST_EUPS_GITREPO:-https://github.com/RobertLuptonTheGood/eups.git}
 # force Python 3
 LSST_PYTHON_VERSION=3
-MINICONDA_VERSION=${MINICONDA_VERSION:-4.3.21}
-MINICONDA_BASE_URL=${MINICONDA_BASE_URL:-https://repo.continuum.io/miniconda}
-CONDA_CHANNELS=${CONDA_CHANNELS:-""}
-GIT_VERSION=${GIT_VERSION:-2.16.1}          # Version of git to install
-LFS_VERSION=${LFS_VERSION:-2.3.4}           # Version of git-lfs to install
+LSST_MINICONDA_VERSION=${LSST_MINICONDA_VERSION:-4.5.4}
```

Comment:
Can you take a look at DM-12214 and close it if it is now irrelevant. DM-12265, DM-12266 and DM-12267 need to be looked at as well. They were all listed as blockers for Anaconda 5.0

Reply:
I had forgotten about the RFC and the related tickets.  I'm linking everything to DM-14011 but can't say much until I've made it through a full a test build... hopefully overnight.